src/neptune_mlflow_plugin/impl/__init__.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2023, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+__all__ = ["create_neptune_tracking_uri"]
+
+from typing import Optional
+from urllib.parse import urlunsplit
+
+PLUGIN_SCHEME = "neptune-plugin"
+
+
+def create_neptune_tracking_uri(project: Optional[str] = None, **kwargs) -> str:
```

Comment:
Have you considered using JSON & base64 for configuration encoding?

Something like `neptune://track/eyJwcm9qZWN0IjogInRlc3RpbmcifQ==`?